web/src/widgetize.ts

```diff
@@ -107,16 +106,16 @@ export function activate(in_opts: WidgetOptions): void {
     }
 }
 
-export function set_widgets_for_list() {
+export function set_widgets_for_list(): void {
     for (const [idx, $widget_elem] of widget_contents) {
-        if (message_lists.current.get(idx) !== undefined) {
-            const $row = message_lists.current.get_row(idx);
+        if (message_lists.current!.get(idx) !== undefined) {
```

Comment:
I agree with  the `assert` idea. But do you think that `const message_list = message_list.current;` will have any extra benefit ? Because `message_list.current` is not an function call or something like that which will require extra computation. Please correct me if I'm getting it wrong. thanks

Reply:
`message_list.current` is not a call ...Agreed. I suggested this for readability purpose. But I don't think it won't make a large difference ,just `asserting` this upfront was important.